src/axolotl/utils/data/rl.py

```diff
@@ -6,9 +6,9 @@
 from pathlib import Path
 from typing import Any, List
 
-import yaml
 from datasets import DatasetDict, concatenate_datasets, load_dataset, load_from_disk
 
+import yaml
```

Comment:
You might need to install `pyyaml` in your local environment and re-run `pre-commit run --all-files`, since these imports should have been correct already.